doc/tutorials/python-kdb.md

```diff
@@ -0,0 +1,39 @@
+# How-To: Python kdb
+
+## Introduction
+
+When programming in python it is possible to access the kdb database, changing values of existing keys, adding and deleting keys and a few other things.
+
+## First Steps
+
+In order to being able to use kdb, obviously, you at first need to `import kdb`. You need access to an python-object of kdb. This is accomlished by calling `kdb.KDB()` and saving this to a variable because later on this object will be needed for various operations.
+
+## Keyset
+
+A keyset is basically a list of the keys that lie within the specified range. At first it is necessairy to create a new keyset. When simply calling `kdb.KeySet()` the keyset is of undefined size. It is possible to specify a certain (maximum) size for a keyset. To load keys into to keyset from the database you simply call the method `get` provided by the kdb-object.
+
+It is also possible to iterate completely normally over a keyset and use len and reversed, dup and deepcopy. The elements of a keyset can be accessed by indexes and a keyset can be sliced. Another way of accessing a key is by the key-name (`keyset_name['path/to/keys/key_name']`). If the key-name does not exist within the keyset, a KeyError exception is thrown.
```

Comment:
Instead of "normally", prefer something like "as expected". Can you bring an example nevertheless for non-python people?